src/data/mockData.js

```diff
@@ -0,0 +1,34 @@
+const products_info = [
```

Comment:
상수데이터의 경우 대문자 + snake_case 활용해주세요!
```suggestion
const PRODUCTS_INFO = [
```

Reply:
오 이런 무의식적으로 소문자로 적었네요..
목 데이터는 이제 필요 없을 것 같아서 아예 삭제했습니다!